# Co-Option and De Novo Gene Evolution Underlie Molluscan Shell Diversity - Supplementary Data

This GitHub repository provides the supplementary data referenced in the publication cited below.

## How to use and cite these files 

All files are publicly available and can be used for further research or other applications. However, if you utilize these resources in your work, we kindly request that you cite our original publication.

**Co-Option and De Novo Gene Evolution Underlie Molluscan Shell Diversity.** Felipe Aguilera, Carmel McDougall, Bernard M. Degnan. *Molecular Biology and Evolution* **34**:779-792 (2017). https://doi.org/10.1093/molbev/msw294.

**Abstract**

Molluscs fabricate shells of incredible diversity and complexity by localized secretions from the dorsal epithelium of the mantle. Although distantly related molluscs express remarkably different secreted gene products, it remains unclear if the evolution of shell structure and pattern is underpinned by the differential co-option of conserved genes or the integration of lineage-specific genes into the mantle regulatory program. To address this, we compare the mantle transcriptomes of 11 bivalves and gastropods of varying relatedness. We find that each species, including four Pinctada (pearl oyster) species that diverged within the last 20 Ma, expresses a unique mantle secretome. Lineage- or species-specific genes comprise a large proportion of each species’ mantle secretome. A majority of these secreted proteins have unique domain architectures that include repetitive, low complexity domains (RLCDs), which evolve rapidly, and have a proclivity to expand, contract and rearrange in the genome. There are also a large number of secretome genes expressed in the mantle that arose before the origin of gastropods and bivalves. Each species expresses a unique set of these more ancient genes consistent with their independent co-option into these mantle gene regulatory networks. From this analysis, we infer lineage-specific secretomes underlie shell diversity, and include both rapidly evolving RLCD-containing proteins, and the continual recruitment and loss of both ancient and recently evolved genes into the periphery of the regulatory network controlling gene expression in the mantle epithelium.

## Author contact

- [Felipe Aguilera](mailto:f.aguilera@uq.edu.au) (first author - PhD student)
- [Carmel McDougall](mailto:c.mcdougall@uq.edu.au) (senior author - PostDoc)
- [Bernie Degnan](b.degnan@uq.edu.au) (corresponding author)
- Lab website: https://www.aguileralab.com

## How to download data files

The files in this repository are ready for use. Simply click on the file, and you will be directed to the GitHub webpage to save it to your device.

## Index of data file contents

[01-Supplementary-Figures.pdf](https://github.com/faguil/Molluscan-Shell-Evolution/blob/main/01-Supplementary-Figures.pdf) contains supplementary figures used in this study.

- **Figure S1.** Computational predictions of cytosolic, membrane-bound and secreted proteins for each species.
- **Figure S2.** GO annotations of genes that arose along the stems leading to bilaterian (PS7, red) and molluscan (PS10, blue) last common ancestors.
- **Figure S3.** Phylogenetic relationships among bivalve and gastropod lineages based on 122 gene families that encode secreted proteins.
- **Figure S4.** GO term (biological process and molecular function) enrichment in newly gained secreted gene families across conchiferan evolution.
- **Figure S5.** Expression profiles of genes encoding secreted proteins – the mantle secretome – of eight bivalve and two gastropod species.
- **Figure S6.** Expression levels of co-opted, lineage-specific and species-specific genes encoding secreted proteins.
- **Figure S7.** Phylogenetic analysis of carbonic anhydrase domain-containing proteins.
- **Figure S8.** Phylogenetic analyses of SPARC domain-containing proteins and zona pellucida-like domain-containing proteins.
- **Figure S9.** Phylogenetic analysis of tyrosinase domain-containing proteins.
- **Figure S10.** Phylogenetic analysis of glycoside hydrolase family 18 domain-containing proteins.
- **Figure S11.** Phylogenetic analyses of polysaccharide deacetylase domain-containing proteins and chitin-binding domain-containing proteins.
- **Figure S12.** Changes in the prediction of gene families using different MCL inflation values.

[02-Supplementary-Tables.xlsx](https://github.com/faguil/Molluscan-Shell-Evolution/blob/main/02-Supplementary-Tables.xlsx) contains supplementary tables used in this study.

- **Table S1.** Results of BLASTP comparisons between shell proteomes and mantle secretomes.
- **Table S2.** Summary of the conchiferan phylostratigraphic analysis.
- **Table S3.** Rate of gene family gain and loss across conchiferan evolution.
- **Table S4.** Enrichment of protein domains present in newly gained gene families over conchiferan evolution.
- **Table S5.** Over-represented GO terms (biological process and molecular function categories) across conchiferan evolution.
- **Table S6.** Functional annotation of gene families across conchiferan evolution.
- **Table S7.** List of lineage- and species-specific gene families across conchiferan evolution.
- **Table S8.** List of gene families with repetive low-complexity domains (RLCDs) across conchiferan evolution.
- **Table S9.** Over-representation of InterPro domains in each phylostrata for each species.
- **Table S10.** Conchiferan mantle sampling used in this study.
- **Table S11.**  Sequence statistics of the de novo assemblies.
- **Table S12.** Clustering, filtering and prediction of open reading frames (ORFs).
- **Table S13.** Contents of the databases used in the BLASTP and TBLASTN sequence similarity searches for each target species.
- **Table S14.** Taxon stability index, leaf stability index, and percentage of missing data for phylogenomic inference of molluscan relationships.
